integration-test/src/integration-test/java/uk/gov/di/ipv/core/integrationtest/DataStoreIpvSessionIT.java

```diff
@@ -1,20 +1,17 @@
 package uk.gov.di.ipv.core.integrationtest;
 
-import com.amazonaws.services.dynamodbv2.AmazonDynamoDB;
```

Comment:
Looking at our GHA's, I don't think we actually run these integration tests anywhere. And I suspect they're a little bit redundant.

Should we just get rid of the integration-test project completely?

Reply:
kill-it-with-fire.gif